kodilla-testing/src/main/java/com/kodilla/testing/weather/stub/WeatherForecast.java

```diff
@@ -13,13 +13,65 @@ public WeatherForecast(Temperatures temperatures) {
     public Map<String, Double> calculateForecast() {
         Map<String, Double> resultMap = new HashMap<>();
 
-        for (Map.Entry<String, Double> temperature :
-                temperatures.getTemperatures().entrySet()) {
-
-            // adding 1 celsius degree to current value
-            // as a temporary weather forecast
-            resultMap.put(temperature.getKey(), temperature.getValue() + 1.0); // [1]
+        for (Map.Entry<String, Double> temperature : temperatures.getTemperatures().entrySet()) {
```

Comment:
Co to jest?

Reply:
to jest for do typu Entry. Nic nie zmieniałem. Ta metoda była podana w sub module. A co jest nie tak?